### PR TITLE
Small modification to wheneverize

### PR DESCRIPTION
**Pull request by advectus**
Create the 'config' directory if directory is not present in project

### CODE REVIEW

File: bin/wheneverize [hunk 1]
```diff
@@ -54,13 +54,16 @@ file = 'config/schedule.rb'
 base = ARGV.empty? ? '.' : ARGV.shift
 
 file = File.join(base, file)
+dir = File.dirname(file)
```
Comment (benlangfeld) [inline bin/wheneverize]: This fits better below with the code that uses it :)

**Comment by benlangfeld** [comment]
Thanks for this contribution. Just one minor nit, but happy to merge this soon!

File: bin/wheneverize (hunk 1 above)
Comment (advectus) [inline bin/wheneverize]: Ok, will add another commit - maybe we can squash the two

**Comment by advectus** [comment]
@benlangfeld re: entry in the CHANGELOG - should I create the new '0.10.0' entry as I don't currently see it

**Comment by benlangfeld** [comment]
> @benlangfeld re: entry in the CHANGELOG - should I create the new '0.10.0' entry as I don't currently see it

No, under the `develop` (should be `master`, really) header please :)

**Comment by benlangfeld** [comment]
Thanks @advectus. Will merge once Travis confirms :)

**Pull request merged**